docs/manual/creating-a-checker.tex

```diff
@@ -1725,7 +1727,7 @@
 \begin{itemize}
 
 \item \code{-Aignorejdkastub}:
-  ignore the \<jdk.astub> file in the checker directory. Files passed
+  ignore the \<jdk.astub> or \<jdkN.astub> file in the checker directory. Files passed
```

Comment:
Should the `or` be an `and`? If there are three files `jdk.astub`, `jdk8.astub`, and `jdk11.astub`, are all three ignored? The sentence could use some tweaking to account for there being only a `jdk8.astub` that should be ignored (but there is no `jdk.astub`).